lib/tyrant/mailer/cell/reset_email.rb

```diff
@@ -0,0 +1,27 @@
+require 'trailblazer/cell'
+require 'cells-slim'
+require 'pathname'
+require 'action_view'
+
+module Tyrant::Cell
+  class ResetEmail < Trailblazer::Cell
+    include ::Cell::Slim
+    include ::ActionView::Helpers::UrlHelper
+
+    self.view_paths << Pathname(__FILE__).ascend{|d| d; break d if d.split.last.to_s == "mailer"}
```

Comment:
Quick loop to get the correct folder path for the view...I know it's not the best but I couldn't think anything smarter than that!